test/test_evaluation/test_test_evaluator.py

```diff
@@ -0,0 +1,206 @@
+import multiprocessing
+import os
+import queue
+import shutil
+import sys
+import unittest
+import unittest.mock
+
+from ConfigSpace import Configuration
+
+import numpy as np
+
+from sklearn.base import BaseEstimator
+
+from smac.tae import StatusType
+
+from autoPyTorch.automl_common.common.utils.backend import create
+from autoPyTorch.datasets.resampling_strategy import NoResamplingStrategyTypes
+from autoPyTorch.evaluation.test_evaluator import TestEvaluator
+from autoPyTorch.evaluation.utils import read_queue
+from autoPyTorch.pipeline.base_pipeline import BasePipeline
+from autoPyTorch.pipeline.components.training.metrics.metrics import accuracy
+
+this_directory = os.path.dirname(__file__)
+sys.path.append(this_directory)
+from evaluation_util import (  # noqa (E402: module level import not at top of file)
+    BaseEvaluatorTest,
+    get_binary_classification_datamanager,
+    get_multiclass_classification_datamanager,
+    get_regression_datamanager,
+)  # noqa (E402: module level import not at top of file)
+
+
+class BackendMock(object):
+    def load_datamanager(self):
+        return get_multiclass_classification_datamanager()
+
+
+class Dummy(object):
+    def __init__(self):
```

Comment:
Those parts are identical to the test_train_evlauator.py, so can we merge this file to the other?
I checked the TestTestEvaluator() and I think this is hard to be merged, so you can just add the lines after L43 directly into test_train_evaluator.py.